src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -2667,6 +2822,59 @@ static void Main()
             CompileAndVerify(comp, expectedOutput: @"System.Byte[]").VerifyDiagnostics();
         }
 
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void OverloadResolution_15()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        System.Console.WriteLine(Test((""s"", 1)));
+    }
+
+    static string Test((object, int) a) => ""object"";
+    static string Test((byte[], int) a) => ""array"";
+}
+";
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+
+            // PROTOTYPE(UTF8StringLiterals) : Add an entry in "docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md"?
+            CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics();
```

Comment:
>CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics(); [](http://example.com/codeflow?start=12&length=69)

Since this is a change in behavior, please include a corresponding compilation using `TestOptions.Regular10`.